src/features/courses/course-controller.ts

```diff
@@ -1216,6 +1216,11 @@ class CourseController {
             calculatedRendererParameters.outputformat = OutputFormat.STATIC;
         }
 
+        // I do not like hardcoding the topicTypeId...
```

Comment:
This might've been a merge that should've been deleted

Reply:
@red-tt-bear @doombot-exe Can I get a confirmation before I remove it?

Reply:
Not sure, this was @doombot-exe s code

Reply:
Though if it's coming as an addition I would tend to agree

Reply:
I moved it into another block